docs/usage.md

```diff
@@ -187,28 +188,29 @@ params {
 }
 ```
 
-### `--fasta1`
+### `--genome2` (using iGenomes)
```

Comment:
I would recommend putting the FASTA flag descriptions first, these will be more often avaliable by most users vs iGenomes. Also you say 'ALSO supports iGenomes' which would also suggest should come second. If you want to push users towards iGenomes, maybe describe what it is and what teh benefit is.

Reply:
You should also describe why you have two flags. I don't yet see a clear description of wht the pipeline is actually _doing_.

Reply:
Added description in REAMDE with a6ddc48